kv/dist_sender.go

```diff
@@ -44,9 +44,13 @@ const (
 	retryBackoff           = 1 * time.Second
 	maxRetryBackoff        = 30 * time.Second
 
-	// Maximum number of ranges to return from an internal range lookup.
-	// TODO(mrtracy): This value should be configurable.
-	rangeLookupMaxRanges = 8
+	// The default maximum number of ranges to return
+	// from an internal range lookup.
+	defaultRangeLookupMaxRanges = 8
+	// The default size of the leader cache.
+	defaultLeaderCacheSize = 1 << 20
```

Comment:
64 entries seems way too small; in general we should probably set this higher than most applications will need (unless doing so would consume tons of memory, but we can easily handle thousands of ranges here).